coverage/src/test/java/gov/cms/ab2d/coverage/util/AB2DPostgresqlContainer.java

```diff
@@ -0,0 +1,27 @@
+package gov.cms.ab2d.coverage.util;
+
+import java.util.Collections;
+import org.testcontainers.containers.PostgreSQLContainer;
+
+public class AB2DPostgresqlContainer extends PostgreSQLContainer<AB2DPostgresqlContainer> {
```

Comment:
What do you think about renaming this class to something like AB2DCoveragePostgressqlContainer in order to disambiguate from other copies?